realhf/impl/model/parallelism/model_parallel/modules.py

```diff
@@ -919,6 +912,7 @@ def __init__(
         output_size,
         bias=True,
         input_is_parallel=True,
+        sequence_parallel=False,
```

Comment:
Why as an argument instead of a runtime option, e.g., `constants.sequence_parallel()`? We may want to enable sequence parallel during training but disable it during inference or generation.